Cubical/Experiments/DunceEquiv.agda

```diff
@@ -0,0 +1,146 @@
+{-
```

Comment:
I don't think we need this attempt as the other proof is so slick. But this file contains some generally useful lemmas that should be moved elsewhere.